### PR TITLE
Feature/lg 520/landing page header contentful

### PR DESCRIPTION
**Pull request by jahlela**
**Description**
- Moves `HomePageHeader` to its own component file
- Query Contentful for new home page header content
- Cycle through Contentful entries to run experiments

### CODE REVIEW

File: src/components/HomePageHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    title: 'titleDe',
+    subtitle: 'subtitleDe'
+  },
+  fr: {
+    title: 'titleFr',
+    subtitle: 'subtitleFr'
+  }
+};
+
+const getExperiment = (experiments, lang) => {
```
[the remark this answers is not in the document]
Comment (jahlela) [reply src/components/HomePageHeader.jsx]: Sure

File: src/pages/index.jsx [hunk 2]
```diff
@@ -1,103 +1,21 @@
...
-            <div id="tablet-ledgy" data-aos="fade-up">
-              <Img {...data.tablet} alt={i18n.t`Screenshot of the Ledgy app`} />
-            </div>
-          </div>
-        </div>
-      </div>
-    </header>
-  );
-};
+import { HomePageHeader } from '../components/HomePageHeader';
```
[the remark this answers is not in the document]
Comment (jahlela) [reply src/pages/index.jsx]: Seemed like it was about time

File: src/components/HomePageHeader.jsx [hunk 3]
```diff
@@ -10,26 +10,38 @@ import { demoUrl, targetBlank, appUrl, trackSignup, isBrowser } from '../layouts
...
+    subtitleKey: 'subtitleFr'
   }
 };
 
-const getExperiment = (experiments, lang) => {
+type ExperimentKeys = {
+  name: string,
+  titleKey: string,
+  subtitleKey: Node
+};
```
Comment (colmarius) [inline src/components/HomePageHeader.jsx]: I think this is now a bit confusing... we should stick to title, subtitle in both types.

Why is subtitleKey here a node? bellow I see it used as a string ...

```
{
  ...,
  subtitle: sampledExperiment[subtitleKey]
}
```

Comment (jahlela) [reply src/components/HomePageHeader.jsx]: Ah, yes, this was supposed to be a string

**Pull request merged**